<launch>
    <include file="$(find speech_to_text)/launch/speech_to_text.launch"/>
    <node name="DoorChecker" pkg="utility" type="DummyDoorCheck.py" output="screen"/>

    <include file="$(find yolov8_ros)/launch/tracker_clean_up_the_table.launch"/>
    <!-- <include file="$(find action_server)/launch/manip_action_servers.launch"/> -->

    <node name="manip_client" pkg="action_client" type="manip_client_smach.py" />
    <node name="navigation_client" pkg="action_client" type="navigation_client_smach.py"/>

    <include file="$(find world_model_kb)/launch/wm_manager.launch"/>

    <node name="clean_the_table_controller" pkg="clean_the_table" type="clean_the_table_controller_smach.py" output="screen">
        <param name="action_topic" type="str" value="/actions"/>
        <param name="action_results_topic" type="str" value="/actions/result"/>
        <param name="vision_topic" type="str" value="/unsw_vision/detections/objects/positions"/>
    </node>
    
</launch>
